.gitops/charts/traffic-court-dev-values.yaml

```diff
@@ -7,14 +7,14 @@ global:
 
 arc-dispute-api:
   image:
-    tag: "1.44.6"
+    tag: "1.45.0"
```

Comment:
In the future, I would recommend that code changes specific to deployment be separate from application changes. This way we can revert commits if necessary and not affect deployment versioning.

Reply:
yes